docs/gettingstarted/index.rst

```diff
@@ -189,14 +191,16 @@ Getting started with Flocker
 
          Simulate a Flocker cluster with virtual machines on your laptop (requires `Vagrant <http://www.vagrantup.com/downloads>`_, `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_):
 
-         .. code-block:: console
+         :version-download:`nodes/Vagrantfile.template`
 
-            you@laptop:~$ git clone \
-              https://github.com/clusterhq/vagrant-flocker && \
-              cd vagrant-flocker && \
-              vagrant up && \
-              [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent) && \
-              ssh-add ~/.vagrant.d/insecure_private_key
+         .. version-code-block:: console
+
+            mkdir vagrant-flocker
+            cd vagrant-flocker
+            wget https://docs.clusterhq.com/en/|latest-installable|/gettingstarted/nodes/Vagrantfile
```

Comment:
Instead of generating the file, perhaps we can use the release branch (fragile?), e.g. `$ wget https://raw.githubusercontent.com/ClusterHQ/flocker/release/flocker-|latest-installable|/README.rst`, but with a path to the Vagrantfile instead of the README.